File: Android/Sources/UserControlLibraries/GaodeMaps/src/main/java/com/genexus/android/controls/maps/gaode/GxMapViewItem.java
package com.genexus.android.controls.maps.gaode;

import com.genexus.android.core.base.model.Entity;
import com.genexus.android.core.controls.maps.common.MapItemBase;
import com.genexus.android.core.controls.maps.common.MapLayer;

import java.util.List;

class GxMapViewItem extends MapItemBase<MapLocation> {
	protected GxMapViewItem(List<MapLocation> locationList, Entity itemData, MapLayer.FeatureType featureType,
	                        String layerId, String themeClassName) {
		super(locationList, itemData, featureType, layerId, themeClassName);
	}
}

File: Android/Sources/UserControlLibraries/HMSMapKitLib/src/main/java/com/genexus/android/controls/maps/hms/GxMapViewItem.kt
package com.genexus.android.controls.maps.hms

import com.genexus.android.core.base.model.Entity
import com.genexus.android.core.controls.maps.common.MapItemBase
import com.genexus.android.core.controls.maps.common.MapLayer.FeatureType

internal class GxMapViewItem(
    locationList: List<MapLocation?>?,
    itemData: Entity?,
    featureType: FeatureType?,
    layerId: String?,
    themeClassName: String?
) : MapItemBase<MapLocation>(locationList, itemData, featureType, layerId, themeClassName)
